consent/migrations/sql/mysql/14.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
```

Comment:
You can put this into `13.sql` and delete the file itself! The same for postgres :)

Reply:
Got it! Thanks.